recipes/flake8-rst/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "flake8-rst" %}
+{% set version = "0.4.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: ec9c7b7a5475c0b3ce8aee4db951f33beffad68bdb79d99d45788d4aa083ce52
+
+build:
+  number: 0
+  entry_points:
+    - flake8-rst = flake8_rst.cli:main
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv "
+
+requirements:
+  host:
+    - flake8 >=3.5.0
+    - pip
+    - python
+  run:
+    - flake8 >=3.5.0
+    - python
+
+test:
+  imports:
+    - flake8_rst
+  commands:
+    - flake8-rst --help
+
+about:
+  home: https://github.com/kataev/flake8-rst
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE.txt
```

Comment:
Please raise an issue about the `license_file` missing from the source distribution upstream and put a link to the issue as a comment in the `meta.yaml` as a reminder for future releases.